dockers/docker-orchagent/critical_processes

```diff
@@ -0,0 +1,11 @@
+orchagent
+portsyncd
+neighsyncd
+vlanmgrd
+intfmgrd
+portmgrd
+buffermgrd
+vrfmgrd
+nbrmgrd
+vxlanmgrd
+intfsyncd
```

Comment:
intfsyncd is no longer present.

Reply:
Thanks! I meant to delete that line, but forgot. It won't cause any issues, but I'll open a new PR to remove it soon.

Reply:
PR here: https://github.com/Azure/sonic-buildimage/pull/2850